lib/ddtrace/contrib/rest_client/request_patch.rb

```diff
@@ -0,0 +1,84 @@
+require 'ddtrace/ext/net'
+require 'ddtrace/ext/distributed'
+
+module Datadog
+  module Contrib
+    module RestClient
+      # RestClient RequestPatch
+      module RequestPatch
+        REQUEST_TRACE_NAME = 'rest_client.request'.freeze
+
+        def self.included(base)
+          base.prepend(InstanceMethods)
```

Comment:
What version of Ruby is supported? `prepend` is not available in 1.9.3, and this will fail for 2.0 because it's a call to a private method. If we plan on supporting 2.0+, we should change this to `send(:prepend, InstanceMethods)`.

We should also add some kind of compatibility check before we reach this point, presumably in the patcher class.

Reply:
Added the shim to support ruby 1.9